spec/requests/api/v1/authentication_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe 'Auth', type: :request do
+
+  describe 'POST /api/v1/auth/sign_in' do
+
+    let(:headers) { { HTTP_ACCEPT: 'application/json' } }
+    let!(:user) { FactoryGirl.create(:user) }
+    let(:credentials) { user.create_new_auth_token }
```

Comment:
Is this used? I can't see `credentials` being used and with `let` it will be lazy evaluated so only when you actually make use of this variable will the code bu run. Did you guys made some research about the difference between `let` vs `let!`